### ISSUE TITLE
Not working with "nova-dependency-container"

### ISSUE THREAD

**Issue by YannikFirre** [manmohanjit/nova-belongs-to-dependency#3]
Hello,

I use your plugin inside a container (with the plugin https://github.com/epartment/nova-dependency-container ) but it's not working.

I try outside it's working :

> BelongsTo::make('Event')->sortable()->rules('required'), 
> BelongsToDependency::make('Page')->dependsOn('event', 'event_id'), 

I try outside it's not working :

> BelongsTo::make('Event') ->sortable()->rules('required'), 
> Select::make('Lien', 'type')->options([ 1 => 'Externe', 2 => 'Page', 3 => 'Bloc', ])->displayUsingLabels()->rules('required'), 
> NovaDependencyContainer::make([ 
>         BelongsToDependency::make('Page')->dependsOn('event', 'event_id')
> ])->dependsOn('type', 2),

Any idea how to do that ?

Thanks a lot :)

**Comment by manmohanjit** [comment]
Hello @InfinityWebMe,

Are you sure you are using NovaDependencyContainer field correctly?
https://github.com/epartment/nova-dependency-container#usage

Make sure to use the trait in the Nova resource:
```use HasDependencies;```

**Comment by YannikFirre** [comment]
@manmohanjit  Thanks, Yes i use "HasDependencies" and i think i use correctly the package.

The package NovaDependencyContainer works well but the field inside contains all items regardless
the dependsOn('event', 'event_id').

here my function fields : 

```
    use HasDependencies;
    use SortsIndexEntries;

    /**
     * Get the fields displayed by the resource.
     *
     * @param  \Illuminate\Http\Request  $request
     * @return array
     */
    public function fields(Request $request)
    {
        return [
            Sortable::make('Ordre', 'id')->onlyOnIndex(),
            BelongsTo::make('Event')
                ->sortable()
                ->rules('required'),
            Text::make('Titre', 'title')->rules('required'),
            Select::make('Lien', 'type')->options([
                1  => 'Externe',
                2 => 'Page',
                3 => 'Bloc',
            ])->displayUsingLabels()->rules('required'),
            NovaDependencyContainer::make([
                Text::make('Lien', 'link'),
            ])->dependsOn('type', 1),
            NovaDependencyContainer::make([
                  BelongsToDependency::make('Page')->dependsOn('event', 'event_id')
            ])->dependsOn('type', 2),
            NovaDependencyContainer::make([
                BelongsTo::make('Bloc', 'block', Block::class)->nullable(),
            ])->dependsOn('type', 3),
            Boolean::make('Active')
                ->trueValue(1)
                ->falseValue(0),
        ];
    }
```

**Comment by manmohanjit** [comment]
Looks like you're right, hmm. I'll try to figure this out.

Feel free to PR if you find a fix before me.

Thanks!

**Comment by manmohanjit** [comment]
I've fixed this in the new release (1.1.0)
